recipes-core/images/seapath-host-efi-test-swu-image.bb

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2021, RTE (http://www.rte-france.com)
```

Comment:
Why RTE copyright ?

Reply:
Because most of this recipe is taken from the seapath-host-efi-swu-image.bb recipe, which belongs to RTE. Maybe I could delete it, but I'm not certain.

Reply:
Ack